lib/game.rb

```diff
@@ -0,0 +1,159 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+   [0,1,2],
+   [3,4,5],
+   [6,7,8],
+   [0,3,6],
+   [1,4,7],
+   [2,5,8],
+   [0,4,8],
+   [2,4,6]
+  ]
+
+  # Defaults to a game with 2 human players and a new board
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    self.board = board
+    self.player_1 = player_1
+    self.player_2 = player_2
+  end
+
+  def current_player
```

Comment:
You can take advantage of the turn_count method from the Board class here. 

`@board.turn_count.even? ? @player_1 : @player_2`